packages/Loader/README.md

```diff
@@ -0,0 +1,80 @@
+## Description
+
+Spinner and ProgressBar component generally allow to show graphically the progress status of a task or process.
+
+## Usage
+
+#### Default Bar
+
+```js
+<ProgressBar progress={40} />
+```
+
+#### Custom progress bar with steps and icon
+
+```js
+const ProgressDemo = () => {
+  const [progress, setProgress] = useState(0);
+
+  useEffect(() => {
+    setTimeout(() => setProgress(100), 2000);
+  }, []);
+
+  const handleChange = ({ value: rangeValue }) =>
+    setProgress(Number(rangeValue));
+
+  const steps = [
+    {
+      start: 0,
+      final: 99,
+      color: "#ffc107",
```

Comment:
if we want to keep the styles decoupled from js we shouldn't move this through a css modifier or similar?
Pros: Css modifier would be decoupled
Const: All the instance of progress bar in the app would have the same kind of colors.